static/js/pm_list_dom.js

```diff
@@ -19,13 +19,13 @@ export function keyed_pm_li(convo) {
     };
 }
 
-export function pm_ul(convos) {
+export function pm_ul(nodes) {
     const attrs = [
-        ["class", "expanded_private_messages"],
+        ["class", "expanded-private-messages"],
```

Comment:
Why the rename from `expanded_private_messages` to `expanded-private-messages`?  In any case, I think the variable/class renames here would all be great preparatory commits that can be merged independently of the feature to shrink this diff.

Reply:
Hmm -- this also can surely come out as a prep commit, thanks :)